Source/Lib/MediaConchLib.py

```diff
@@ -16,10 +17,26 @@
 from ctypes import c_long as _c_long
 from ctypes import c_int as _c_int
 
-if _sys.platform == "darwin":
-    _MediaConchLib_Handler = _CDLL("libmediaconch.0.dylib")
+
+
+_module = _os.path.abspath(_os.path.dirname(__file__))
+_machine = _platform.machine().lower()
+_system = _platform.system().lower()
+
+if _machine in ["i386", "i686"]:
+    _machine = "x86_32"
+elif _machine in ["amd64"]:
+    _machine = "x86_64"
+
+if _system == "darwin":

```

Comment:
Win was never supported there? Seems easy to add it, if it is the case please add win `.dll`.